src/models/model.py

```diff
@@ -62,6 +64,10 @@ def _calc_input_channels(self):
         if self.include_right_view:
             self.input_channels = 2 * self.input_channels
 
+    def _log_fid(self, pred, target):
+        fid_val = calculate_fid(pred, target, batch_size=16, device=self.device)
```

Comment:
@jeswan I dont think the batch size should be hard coded here - shouldnt it just take self.batch_size?

Reply:
this batch_size is a little different than the unet batch size. It's the batch size for the Inception network running under the hood of FID

Reply:
does it still work when our batch size is not exactly 16? i.e. 32 or even like 8?

Reply:
removed since batch_size was not used.